src/kernel/captbl.c

```diff
@@ -15,7 +15,7 @@ captbl_activate_boot(struct captbl *t, unsigned long cap)
 	int                ret;
 
 	ctc = (struct cap_captbl *)captbl_add(t, cap, CAP_CAPTBL, &ret);
-	if (!ctc) return ret;
+	if (!ctc) { return ret; }
```

Comment:
This is inconsistent styling. It is ok to have single-line conditionals/loops without `{}`. I we were to require `{}`, then they should be everywhere. I don't htink we need to require it. I'll just make a note of this where I see it in the PR.